plugins/jira/jira.plugin.zsh

```diff
@@ -59,6 +59,16 @@ open_jira_issue () {
   fi
 }
 
+jira_url_help() {
+  cat << EOF
+JIRA url is not specified anywhere.
+Valid options, in order of preference:
```

Comment:
Should this say "precedence" instead of "preference"? This list is the order in which they're checked, not in order of which is "better", right?